pepper-apis/src/main/java/org/broadinstitute/ddp/model/study/StudyLanguage.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.ddp.model.study;
+
+public class StudyLanguage {
+    private String languageCode;
+    private String displayName;
+    private boolean isDefault;
+    private long studyId;
+    private long languageId;
+
+
+    public StudyLanguage(String languageCode, String displayName, boolean isDefault, long studyId, long languageId) {
```

Comment:
I recommend adding annotations to help facilitate JDBI mapping the SQL data properly:
```java
@JdbiConstructor
public StudyLanguage(
    @ColumnName("language_code") String languageCode,
    @ColumnName("name") String displayName,
    ...) {
}
```